blog.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <title>Adoptadog</title>
+</head>
+
+<body>
+
+  <header>
+
+  <img src="https://raw.githubusercontent.com/Sophiagts/html200-adopt-a-dog/41dab5423bd1111155d9d5bd7f6a44c57e8119e2/images/adoptadog-logo.svg" alt="adoptadog logo">
```

Comment:
You will want to use local paths for your images, not the direct links like this.